### PR TITLE
Add missing properties on `Review`

### PR DESCRIPTION
**Pull request by remi-stripe**
r? @ob-stripe 
cc @stripe/api-libraries

### CODE REVIEW

File: review.go [hunk 1]
```diff
@@ -32,17 +52,40 @@ type ReviewListParams struct {
...
+	Longitude float64 `json:"longitude"`
+	Region    string  `json:"region"`
+}
+
+// ReviewSession represents information about the browser session associated with a review.
+type ReviewSession struct {
+	Browser  string `json:"browser"`
+	Device   string `json:"device"`
+	Platform string `json:"platform"`
+	Version  string `json:"session"`
```
Comment (ob-stripe) [inline review.go:69]:
```suggestion
	Version  string `json:"version"`
```

Comment (remi-stripe) [reply review.go:69]: Would you believe me if I said I do this to make sure you review properly? 😓

Comment (ob-stripe) [reply review.go:69]: ಠ_ಠ

**Comment by remi-stripe** [comment]
@ob-stripe Fixed, PTAL

**Pull request merged**